test/e2e/providers/vsphere/vsphere.go

```diff
@@ -69,20 +70,18 @@ func (p *Provider) newVSphereMachineProviderSpec() (*mapi.VSphereMachineProvider
 		CredentialsSecret: &core.LocalObjectReference{
 			Name: defaultCredentialsSecretName,
 		},
-		DiskGiB:   int32(128),
-		MemoryMiB: int64(16384),
-		Network: mapi.NetworkSpec{
-			Devices: []mapi.NetworkDeviceSpec{{NetworkName: getNetwork()}},
-		},
+		DiskGiB:           int32(128),
+		MemoryMiB:         int64(16384),
+		Network:           existingProviderSpecs.Network,
 		NumCPUs:           int32(4),
 		NumCoresPerSocket: int32(1),
 		Template:          vmTemplate,
-		Workspace:         workspace,
+		Workspace:         existingProviderSpecs.Workspace,
 	}, nil
 }
 
-// getWorkspaceFromExistingMachineSet returns Workspace from a machineset provisioned during installation
-func (p *Provider) getWorkspaceFromExistingMachineSet() (*mapi.Workspace, error) {
+// getSpecsFromExistingMachineSet returns the providerSpecs of an existing machineset provisioned during installation
+func (p *Provider) getSpecsFromExistingMachineSet() (*mapi.VSphereMachineProviderSpec, error) {
```

Comment:
Please name this ` getProviderSpecFromExistingMachineSet()`.

Reply:
Addressed.